LayoutTests/platform/glib/damage/common.js

```diff
@@ -100,19 +121,40 @@ function log(entity) {
     console.log(JSON.stringify(entity));
 }
 
+function createNewElement(elementName, lambda = (el) => {}) {
+    var newElement = document.createElement(elementName);
+    lambda(newElement);
+    return newElement;
+}
+
 function createNewElementWithClass(elementName, className, lambda = (el) => {}) {
     var newElement = document.createElement(elementName);
     newElement.className = className;
     lambda(newElement);
     return newElement;
 }
```

Comment:
```suggestion
function createNewElementWithClass(elementName, className, lambda = (el) => {}) {
    let decoratedLambda = (e, l) => { e.className = className; l(e); };
    return createNewElement(elementName, decoratedLambda);
}
```

I've tried the above locally on the WebInspector and it seems to work, which would allow us to make the `crewNewElementWithClass` and the `spawnNewElementWithClass` a specialisation of the `create` and `spawn` methods... which might not be a great gain. It made more sense when I first started thinking about it than when I see it now :-D

Reply:
let's keep it simple then